lib/tasks/cms/site.rake

```diff
@@ -0,0 +1,38 @@
+namespace :cms do
+  def mock_task(attr)
+    task = OpenStruct.new(attr)
+    def task.log(msg); puts(msg); end
```

Comment:
__[rubocop]__
* Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->